pkg/management/postgres/instance_replica.go

```diff
@@ -64,7 +64,8 @@ func (instance *Instance) RefreshReplicaConfiguration(
 
 func (instance *Instance) writeReplicaConfigurationForReplica(cluster *apiv1.Cluster) (changed bool, err error) {
 	slotName := cluster.GetSlotNameFromInstanceName(instance.GetPodName())
-	return UpdateReplicaConfiguration(instance.PgData, instance.GetPrimaryConnInfo(), slotName)
+	primaryConnInfo := instance.GetPrimaryConnInfo() + fmt.Sprintf(" dbname=%v", cluster.GetApplicationDatabaseName())
```

Comment:
I would use the "postgres" database here because it always exists and does not require any additional logic.

Reply:
thanks! this has been updated